lib/va_profile/profile_information/transaction_response.rb

```diff
@@ -0,0 +1,194 @@
+# frozen_string_literal: true
+
+require 'va_profile/models/transaction'
+require 'va_profile/response'
+
+module VAProfile
+  module ProfileInformation
+    class TransactionResponse < VAProfile::Response
+      extend SentryLogging
+
+      attribute :transaction, VAProfile::Models::Transaction
+      ERROR_STATUS = 'COMPLETED_FAILURE'
+
+      attr_reader :response_body
+
+      def self.from(raw_response = nil, _user = nil)
+        @response_body = raw_response&.body
+        if error?
+          log_message_to_sentry(
+            'VAProfile transaction error',
+            :error,
+            { response_body: @response_body },
```

Comment:
Will response_body have any PII when logging to Sentry?

Reply:
I am not sure. I am assuming not since it's already being sent